cmd/protoc-gen-go-tetragon/common/common.go

```diff
@@ -235,10 +246,24 @@ func GetEvents(f *protogen.File) ([]*protogen.Message, error) {
 
 var fieldsCache []*protogen.Message
 
+var reservedFields = []string{
+	"Timestamp",
+	"UInt32Value",
+}
+
+func isReservedField(field string) bool {
+	for _, f := range reservedFields {
+		if field == f {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Small nit, a map would be more idiomatic here I think.

Reply:
yeah its nicer that way.